specification/containerservice/resource-manager/Microsoft.ContainerService/aks/preview/2023-10-02-preview/managedClusters.json

```diff
@@ -2556,6 +2556,51 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ContainerService/managedClusters/{resourceName}/agentPools/{agentPoolName}/operations/{operationId}": {
+      "get": {
+        "tags": [
+          "OperationStatusResult"
+        ],
+        "operationId": "OperationStatusResult_GetByAgentPool",
+        "summary": "Get the status of a specific operation in the specified managed cluster.",
```

Comment:
```suggestion
        "summary": "Get the status of a specific operation in the specified agent pool.",
```

Reply:
I had resolved this problem before making a suggestion.